src/app/domains/Form/routes/FormShow/FormShow.js

```diff
@@ -7,6 +7,8 @@ import PropTypes from 'prop-types'
 import { globalStyles } from 'app/styles'
 import { MiddleContainer } from '~/app/components'
 import { FormAdvancedView } from 'domains/Form/components'
+import { QuestionAdvancedView } from 'domains/Question/components'
+import { LAYOUT_TYPES, QUESTION_TYPES } from 'app/constants'
```

Comment:
**On 62th line:** move styles to separate styles file.

Reply:
fixed